### ISSUE TITLE
cpdb v4.1 interaction doublets

### ISSUE THREAD

**Issue by JanLeipzig** [JanLeipzig/ViennaCCCdb#18]
POMC	OPRM1	P01189	P35372	Liana_v0.1.7;Cpdb_v4;Cpdb_v4
POMC	OPRK1	P01189	P41145	Liana_v0.1.7;Cpdb_v4;Cpdb_v4
POMC	OPRD1	P01189	P41143	Liana_v0.1.7;Cpdb_v4;Cpdb_v4

POMC is listed as individual interaction, as well as with b-Endorphin_byPOMC. Causes doublet

**Comment by dnjst** [comment]
Good catch!  I think these will need to be corrected in the source database and reported it
